src/App.tsx

```diff
@@ -1,109 +1,36 @@
+/* eslint-disable max-len */
 import React from 'react';
-
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
-
-export const App: React.FC = () => (
-  <section className="App">
-    <h1 className="App__title">Static list of posts</h1>
-
-    <div className="PostList">
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">qui est esse</h3>
-
-          <p>
-            {' Posted by  '}
-
-            <a className="UserInfo" href="mailto:Sincere@april.biz">
-              Leanne Graham
-            </a>
-          </p>
-        </div>
-
-        <p className="PostInfo__body">
-          est rerum tempore vitae sequi sint nihil reprehenderit dolor beatae
-          ea dolores neque fugiat blanditiis voluptate porro vel nihil
-          molestiae ut reiciendis qui aperiam non debitis possimus qui neque
-          nisi nulla
-        </p>
-
-        <hr />
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
 
-        <b data-cy="NoCommentsMessage">No comments yet</b>
-      </div>
+import { PostList } from './components/PostList';
 
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">
-            doloremque illum aliquid sunt
-          </h3>
+import { Comment } from './types/Comment';
+import { User } from './types/User';
+import { Post } from './types/Post';
 
-          <p>
-            {' Posted by  '}
+function getCommentsById(postId: number):Comment[] {
+  return commentsFromServer.filter(comment => (postId === comment.postId)) || [];
```

Comment:
filter will return empty object by default if there no comments, no need to specify it